src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -89,9 +93,26 @@ object CassandraLauncher {
    * @throws akka.persistence.cassandra.testkit.CassandraLauncher.CleanFailedException if `clean`
    *   is `true` and removal of the directory fails
    */
-  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit = this.synchronized {
-    if (cassandraDaemon.isEmpty) {
+  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit =
+    start(cassandraDirectory, configResource, clean, port, fork = false)
```

Comment:
I'm not sure. I think we need to discuss deprecation and backwards compatibility in general - the primary use for CassandraLauncher is for tests, and what we've found is that for some reason, running Cassandra in process when running tests from sbt works for most people, but not all, but then it doesn't work in Maven or when running tests from IntelliJ. We haven't worked out why it works in sbt yet, but the point is, if forking is not the default, then every project is going to go through a phase of first not forking, then finding it doesn't work in some environments, wasting time debugging, then finding the fork option and switching to that.  So perhaps forking should be the default?  But that's a big potentially breaking change, and the classpath discovery method may not work in every environment.